File: Core python/Assignment/Assignment 3/q6.py
# Write a program to calculate profit or loss.


# take input of cost price and selling price

cp = float(input("enter cost price : "))
sp = float(input("enter selling price : "))

# print output base on condition
if(sp > cp):
    print(f'profit : {sp-cp}')
elif(cp > sp):
    print(f'loss : {sp-cp}')
else:
    print(f'no profit or loss ')

File: C to Java/Assignments/Assignment 5/q6.c
/*
6. Print a half pyramid using numbers
Input: n = 5
Output:
1
12
123
1234
12345
*/

void main() {
	
	int n ;
	printf("enter n : ");
	scanf("%d", &n);
	
	for(int i = 1 ; i <= n ; i++){
		for(int j=1 ; j <= i ; j++){
			printf("%d ", j);
		}
		printf("\n");
	}
}